.env

```diff
@@ -70,16 +70,18 @@ HASH_TEMPORAL_VISIBILITY_PG_DATABASE=temporal_visibility
 HASH_GRAPH_PG_USER=graph
 HASH_GRAPH_PG_PASSWORD=graph
 HASH_GRAPH_PG_DATABASE=graph
-HASH_GRAPH_LOG_LEVEL=trace,h2=info,tokio_util=debug,tower=info,tonic=debug,hyper=info,tokio_postgres=info,rustls=info,tarpc=warn
+HASH_GRAPH_LOG_LEVEL=info
 
 HASH_GRAPH_TYPE_FETCHER_HOST=localhost
 HASH_GRAPH_TYPE_FETCHER_PORT=4455
 
 HASH_GRAPH_REALTIME_PG_USER=realtime
 HASH_GRAPH_REALTIME_PG_PASSWORD=realtime
 
-HASH_GRAPH_API_HOST=127.0.0.1
-HASH_GRAPH_API_PORT=4000
+HASH_GRAPH_HTTP_HOST=127.0.0.1
+HASH_GRAPH_HTTP_PORT=4000
+HASH_GRAPH_RPC_HOST=127.0.0.1
```

Comment:
are we sure we want to use the same variable for both client and host? considering that the client might have different host to the server (I expect the server to always bind to an IP, but the client to connect over DNS)

Reply:
This is how we handle other environment variables currently as well. In production you typically specify the host per instance anyway, so you can pass different environment variable to different services. E.g. in Docker you'd use the container name of the server for the client and set the server manually to `127.0.0.1`. In AWS you'd sets the host to `0.0.0.0` and (in our case) use the DNS name of the server for the client.